src/gtk/notifmsg.cpp

```diff
@@ -312,12 +312,28 @@ class wxLibNotifyMsgImpl : public wxNotificationMessageImpl
         // Values according to the OpenDesktop specification at:
         // https://developer.gnome.org/notification-spec/
 
+
+        wxNotificationMessage::DismissalReason reason = wxNotificationMessage::DismissalReason::Unknown;
+        switch (closeReason)
+        {
+        case 1: // "The notification expired."
+            reason = wxNotificationMessage::DismissalReason::TimedOut;
+            break;
+        case 2: // "The notification was dismissed by the user."
+            reason = wxNotificationMessage::DismissalReason::DismissedByUser;
+            break;
+        case 3: // "The notification was closed by a call to CloseNotification."
+            reason = wxNotificationMessage::DismissalReason::DismissedByApp;
+            break;
+        }
+
         switch (closeReason)
```

Comment:
Sorry, it doesn't make to have 2 `switch (closeReason)` to me in a row. This should probably be changed to `switch (reason)` now?

Reply:
It makes some sense, as the cases aren't the same in both. But yes, `switch (reason)` for the latter would be better.